### DISCUSSION TITLE
Iris incompatible with latest version of sodium.

### DISCUSSION THREAD

**Discussion by Gelvey** [IrisShaders/Iris#2189] [Q&A] [answered]
![image](https://github.com/IrisShaders/Iris/assets/68216784/aa6e7652-8bf8-4708-8c85-250500c4d1a8)

Please update Iris to support Sodium 0.5.5 (latest).

**Comment by IMS212** [comment, marked as answer]
done

**Reply by Gelvey** [reply]
Works, thanks.
